src/features/Licenses/LicensesPage/index.jsx

```diff
@@ -25,16 +25,16 @@ const LicensesPage = () => {
 
   useEffect(() => {
     if (Object.keys(selectedInstitution).length > 0) {
-      dispatch(fetchLicensesData(selectedInstitution?.id, initialPage));
+      dispatch(fetchLicensesData(selectedInstitution.id, initialPage));
     }
 
-    return () => {
-      dispatch(resetLicensesTable());
-    };
-  }, [selectedInstitution, dispatch]);
+    return () => dispatch(resetLicensesTable());
+  }, [selectedInstitution, dispatch]); // eslint-disable-line react-hooks/exhaustive-deps
 
   useEffect(() => {
-    dispatch(fetchLicensesData(selectedInstitution?.id, currentPage, stateLicenses.filters));
+    if (Object.keys(selectedInstitution).length > 0) {
+      dispatch(fetchLicensesData(selectedInstitution.id, currentPage, stateLicenses.filters));
+    }
```

Comment:
Same comment about to have twice effects for "the same"

Reply:
@01001110J I tried to use only one, but if you apply filters then you go to other page and then come back to License Inventory page, the filters will be applied yet.

So, the second useEffect hook is to avoid that scenario and clean the previous result.

Reply:
https://github.com/Pearson-Advance/frontend-app-institution-portal/pull/138#discussion_r1884217026